src/core/Microsoft.PowerToys.Settings.UI.Lib/Contributor.cs

```diff
@@ -1,12 +0,0 @@
-﻿using System;
```

Comment:
Are we removing contributors list from the UI?

Reply:
We are no longer rendering it dynamically.